README.md

```diff
@@ -113,70 +113,75 @@ $ docker run --name heka -it -p 4352:4352 -p 3242:3242 -v <path to heka-tcp-conf
 Where port 4352 is the Heka dashboard port and 3242 is a sample TCP input port.
 
 
-In one terminal window, start the snap daemon (in this case with logging set to 1 and trust disabled):
-```
-$ $SNAP_PATH/bin/snapd -l 1 -t 0
-```
+Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
 
-In another terminal window:
-Load snap collector plugins. For example:
-```
-$ $SNAP_PATH/bin/snapctl plugin load $SNAP_PSUTIL_PLUGIN/build/rootfs/snap-plugin-collector-psutil
+Ensure [Snap daemon is running](https://github.com/intelsdi-x/snap#running-snap):
+* initd: `service snap-telemetry start`
+* systemd: `systemctl start snap-telemetry`
+* command line: `sudo snapd -l 1 -t 0 &`
+
+Download and load snap-plugin-collector-psutil plugin (path to binary file for Linux/amd64):
 ```
-Load snap heka publisher plugin
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-psutil/latest/linux/x86_64/snap-plugin-collector-psutil
+$ snapctl plugin load snap-plugin-collector-psutil
 ```
-$ $SNAP_PATH/bin/snapctl plugin load <path to built snap-plugin-publisher-heka plugin>
+
+Build heka according to the [instruction](#building-from-source) and go to directory with plugin binary file.
```

Comment:
Consider also adding wget to heka plugin - just in case user just wants to try or use the plugin but not necessarily build the plugin.